tests/ui/invalid-compile-flags/prune-link-args.rs

```diff
@@ -0,0 +1,13 @@
+// Passing link-args with an unexpected space
```

Comment:
This might need the flags to ignoring cross compile and Windows.

Reply:
I restored the "ignore cross compile" flag, but, when I see a Windows ignore flag which is not directly explained, I try to leave it in to see if it still fails compilation on Windows today.